### ISSUE TITLE
Make visible the branches that contain the result of a search

### ISSUE THREAD

**Issue by CheyenneWills** [sublimehq/sublime_merge#1745]
**Problem description**

Provide an option that will make visible the branches that contain the results of a search.  For example, say a search for specific content produces 4 different commits, but they are spread across 2 branches.  have an option that would mark those 2 branches as visible.

**Preferred solution**
A command pallet option, or a clickable menu option would be sufficient

**Alternatives**

N/A

**Screenshots**

N/A

**Comment by tommcc** [comment]
Yes, this is currently my main issue with SM - the search is always missing a ton of context, making it hard to reason about where features are in a large codebase with many developers.

**Comment by themilkman** [comment]
At least it might be helpful to see the containing branches on the commit details as requested here in #1211.
Maybe one could right click on those in the commit details to show/hide them.
